content/python/concepts/strings/terms/zfill/zfill.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: '.zfill()' 
+Description: 'Takes an int and returns a copy of the string with a number of zeros preppended as a suffix , i.e "no. of zeros" + string.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The **`.zfill()`**  method pads a string with zeros on the left to maintain a specific length. It takes an integer argument, and the number of zeros added is determined by the difference between the specified length and the length of the original string.
+
+> **Note:** The `.zfill()` method does not change the string it is used on.
+If no argument is passed into the method. A type error is thrown
```

Comment:
```suggestion
> **Note:** The `.zfill()` method does not change the string it is used on.
```
We don't need the line about the error, just note above that the parameter is required.